images/dnsmasq/validate_dynamic_deps.py

```diff
@@ -0,0 +1,158 @@
+#/usr/bin/env python3
```

Comment:
copyright

is this code copied from somewhere else?

Reply:
Just to confirm, you're saying to add the Kubernetes copyright?

This is original code.

Reply:
Yes, you need the copyright header.

Reply:
Added the copyright header